### DISCUSSION TITLE
Changing RGB data from 8 to 16 bits

### DISCUSSION THREAD

**Discussion by matismatis** [r-lidar/lidR#635] [Q&A]
I have created a point cloud dataset from a forest canopy, saved as a LAS file. I am trying to analyze it using the lidR package on Rstudio, to segmentate individual tree crowns. I know that the package was created for lidar data, but I hope I can also analyze LAS files created with orthography. 
After running the package using my LAS files created by WEBODM, I get several errors.

```r
las <- lidR::readLAS(''lasfiles'')
Warning message:
Invalid data: RGB colors are recorded on 8 bits instead of 16 bits.
```

```
Further errors which I want to resolve later:
Error in plot.window(…) : need finite ‘ylim’ values
In addition: Warning messages:
1: In xy.coords(x, y, xlabel, ylabel, log) : NAs introduced by coercion
2: In min(x) : no non-missing arguments to min; returning Inf
3: In max(x) : no non-missing arguments to max; returning -Inf
```

Does anyone know how to change the RGB colors to 16 bit? Or is there a way to remove colors, to only create point clouds without color data?
Thank you very much, please let me know if I didn’t properly describe my problem.

[LAS dataset used](https://hvhl-my.sharepoint.com/:u:/r/personal/matis_aerts_hvhl_nl/Documents/Attachments/[Overloonsche-Duinen-11-4-2022-georeferenced_model.las](https://hvhl-my.sharepoint.com/:u:/r/personal/matis_aerts_hvhl_nl/Documents/Attachments/Overloonsche-Duinen-11-4-2022-georeferenced_model.las?csf=1&web=1&e=uefdbl)?csf=1&web=1&e=uefdbl)

Best regards,
Matis

**Comment by Jean-Romain** [comment]
You can keep your data 8 bits. But when you are working with these data using lidR data you must pay attention to this fact. Expect `plot(las, color = "RGB")` I don't think there is other function that are processing RGB natively so it is not a big deal.
